site/themes/citra-bs-theme/layouts/shortcodes/audio.html

```diff
@@ -0,0 +1,4 @@
+<audio preload="auto" controls>
+    <source src="{{ index .Params 0 }}" type="audio/ogg">
```

Comment:
For consistency, those should be accessed via a 'src' param, rather than just in the body. For example:
```html
<source src="{{.Get "src"}}" type="audio/ogg">
```

Reply:
I was following the `{{<youtube id>}}` template rather than the `{{<figure src>}}` template for prototype, and it just asks for a video ID.

Reply:
Ah, thats totally fine. Feel like we should be updating the Youtube template to use attr="value" pairs anyway, as we might want to specify additional properties (dimensions, autoplay, etc) down the line.

Reply:
Okay, fair enough! I'll add parameters to this and push ASAP. 😄